packages/react-components/theme-designer/src/components/Demo/Demo.tsx

```diff
@@ -204,13 +202,13 @@ export const Column3 = () => {
 export const Demo: React.FC<ContentProps> = props => {
   const styles = useStyles();
   return (
-    <FluentProvider theme={teamsLightTheme}>
```

Comment:
I take it back. The app will get a static theme (or maybe let users pick light or dark mode) but the Demo is going to get wrapped in the custom theme created by the user. So lets leave this provider in here, and make the theme a prop.